core/trino-main/src/test/java/io/trino/TestSession.java

```diff
@@ -61,4 +69,53 @@ public void testAddSecondCatalogProperty()
                         "first_property", "some_value",
                         "second_property", "another_value")));
     }
+
+    @Test
+    public void testCreateViewSession()
+    {
+        final var catalog = Optional.of("test_catalog");
+        final var schema = Optional.of("test_schema");
+        final var queryId = new QueryId("test_query_id");
+        final var transactionId = TransactionId.create();
+        final var identity = new Identity.Builder("test_user").build();
+        final var originalIdentity = new Identity.Builder("test_original_user").build();
+        final var source = Optional.of("test_source");
+        final var timeZoneKey = TimeZoneKey.UTC_KEY;
+        final var locale = Locale.ENGLISH;
+        final var remoteUserAddress = Optional.of("1.1.1.1");
+        final var userAgent = Optional.of("test_agent");
+        final var clientInfo = Optional.of("test_client_info");
+        final var traceToken = Optional.of("test_trace_token");
+        final var start = Instant.ofEpochMilli(2L);
+
+        Session originalSession = Session.builder(testSessionBuilder().build())
+                .setQueryId(queryId)
+                .setTransactionId(transactionId)
+                .setOriginalIdentity(originalIdentity)
+                .setSource(source)
+                .setTimeZoneKey(timeZoneKey)
+                .setLocale(locale)
+                .setRemoteUserAddress(remoteUserAddress)
+                .setUserAgent(userAgent)
+                .setClientInfo(clientInfo)
+                .setTraceToken(traceToken)
+                .setStart(start)
+                .build();
+
+        Session viewSession = originalSession.createViewSession(catalog, schema, identity,
+                Collections.emptyList());
```

Comment:
`Collections.emptyList()` -> `ImmutableList.of()`
Please keep all arguments on same line, or each argument on new line